sdk/turing/session.py

```diff
@@ -12,8 +14,10 @@
     EnsemblersPaginatedResults, \
     EnsemblingJobPaginatedResults, \
     IdObject, \
+    Router, \
     RouterDetails, \
-    RouterConfig
+    RouterConfig, \
+    RouterVersion
```

Comment:
no action required but you can actually use parenthesis also: https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses